changelog/16723.txt

```diff
@@ -0,0 +1,4 @@
+```release-note:feature
```

Comment:
👋🏻 This changelog entry isn't correctly formatted for new features.

`**OCSP Responder**: <text>`

You can check out the 1.11 changelog for a reference, and there's also a wiki page about this. Could you please update this file on main so that it formats correctly in the final version of the 1.12 changelog?